demo/src/tables.py

```diff
@@ -146,7 +147,14 @@ def tabs() -> list[AnyComponent]:
 
 @router.get('/users/{id}/', response_model=FastUI, response_model_exclude_none=True)
 def user_profile(id: int) -> list[AnyComponent]:
-    user: User | None = users[id - 1] if id <= len(users) else None
+    user = users[id - 1] if id <= len(users) else None
+    if user is None:
+        return demo_page(
+            *tabs(),
+            c.Text(text='User not found.'),
+            title='User not found',
+        )
+
```

Comment:
I'd prefer if we don't have any behavioral changes in this PR - just linting fixes!

Reply:
True, but that's also on demo. It doesn't affect the real package. 
Side note -> this change was made to fix type check issues Otherwise i'd have to use # type: ignore